### PR TITLE
fix(Angular InstantSearch): upgrade extending-widgets example

### PR DESCRIPTION
**Pull request by tkrugg**
Updates extending-widget example.
Related guide has already been updated: https://www.algolia.com/doc/guides/building-search-ui/widgets/customize-an-existing-widget/angular/#customize-the-complete-ui-of-the-widgets

https://github.com/algolia/doc-code-samples/issues/255

### CODE REVIEW

File: Angular InstantSearch/extending-widgets/src/app/app.component.html [hunk 1]
```diff
@@ -1,30 +1,60 @@
...
+        <div class="searchbox">
+          <ais-search-box placeholder=""></ais-search-box>
+        </div>
+        <div class="filters">
+          <ais-menu-select></ais-menu-select>
+        </div>
+        <ais-hits>
+          <ng-template let-hits="hits" let-results="results">
+            <div *ngIf="hits.length === 0">
+              No results found matching <strong></strong>.
```
Comment (Haroenv) [inline Angular InstantSearch/extending-widgets/src/app/app.component.html:26]:
```suggestion
              No results found.
```

we don't have access to the query here

File: Angular InstantSearch/extending-widgets/src/app/app.component.html [hunk 2]
```diff
@@ -1,30 +1,60 @@
...
-          </ol>
-        </ng-template>
-      </ais-hits>
+    <div class="search-panel">
+      <div class="search-panel__results">
+        <div class="searchbox">
+          <ais-search-box placeholder=""></ais-search-box>
+        </div>
+        <div class="filters">
+          <ais-menu-select></ais-menu-select>
```
Comment (Haroenv) [inline Angular InstantSearch/extending-widgets/src/app/app.component.html:21]: I know it's the same as the guide, but we probably should guide people to make their own prefix, not `ais`, right?

**Review by Haroenv — COMMENTED** [review]
widget itself is good, two smaller questions :)

File: Angular InstantSearch/extending-widgets/src/app/app.component.html (hunk 2 above)
Comment (tkrugg) [reply Angular InstantSearch/extending-widgets/src/app/app.component.html:21]: That's an excellent point. I'll use `my-` prefix, WDYT ?

Comment (Haroenv) [reply Angular InstantSearch/extending-widgets/src/app/app.component.html:21]: my- sounds good

**Review by Haroenv — APPROVED** [review]
obviously the guide also needs updating now :)

**Comment by tkrugg** [comment]
@Haroenv 

> obviously the guide also needs updating now :)

Done here https://github.com/algolia/doc/pull/6430

**Pull request merged**